app/__init__.py

```diff
@@ -34,6 +31,16 @@ def create_app(config_name):
         search_api_client=search_api_client
     )
 
+    for framework_data in data_api_client.find_frameworks().get('frameworks'):
```

Comment:
Yeah, hopefully mitigated by the fact that we're loading up 'open' and 'standstill' frameworks too - chances are we'll have had at least one restart between the new framework first appearing (e.g. becoming open) and it actually going into the 'live' state.

Dunno that applies to DOS frameworks, mind.

I'm still slightly worried about the long-term implications for memory usage seeing as each framework a megabyte or so of data. Maybe that's actually peanuts, on reflection :)